etl/steps/data/garden/emdat/2024-04-11/natural_disasters.meta.yml

```diff
@@ -1,48 +1,81 @@
+  definition_earthquake: &definition-earthquake |-
+    An earthquake is defined as a sudden movement of a block of the Earth's crust along a geological fault and associated ground shaking.
+  definition_extreme_temperature: &definition-extreme-temperature |-
+    Extreme temperature is used as a general term for temperature variations above (extreme heat) or below (extreme cold) normal conditions.
+  definition_flood: &definition-flood |-
+    Flood is used as a general term for the overflow of water from a stream channel onto normally dry land in the floodplain (riverine flooding), higher-than-normal levels along the coast (coastal flooding) and in lakes or reservoirs as well as ponding of water at or near the point where the rain fell (flash floods).
+  # definition_landslide: &definition-landslide |-
+  #   EM-DAT defines a landslide as any kind of moderate to rapid soil movement including lahars, mudslides, and debris flows (under dry conditions). A landslide is the movement of soil or rock controlled by gravity and the speed of the movement usually ranges between slow and rapid, but it is not very slow. It can be superficial or deep, but the materials must make up a mass that is a portion of the slope or the slope itself. The movement has to be downward and outward with a free face.
+  # definition_storm: &definition-storm |-
+  #   EM-DAT uses 'storm' as a general term for a disturbance in the atmosphere that results in strong winds accompanied by rain, snow, or other precipitation.
+  definition_volcanic_eruption: &definition-volcanic-eruption |-
+    Volcanic activity is defined as any type of volcanic event near an opening/vent in the Earth's surface including volcanic eruptions of lava, ash, hot vapor, gas, and pyroclastic material.
+  definition_wildfire: &definition-wildfire |-
+    A wildfire is defined as any uncontrolled and non-prescribed combustion or burning of plants in a natural setting such as a forest, grassland, brush land or tundra, which consumes natural fuels and spreads based on environmental conditions (e.g., wind, or topography). Wildfires can be triggered by lightning or human actions.
+  definition_dry_mass_movement: &definition-dry-mass-movement |-
+    A dry mass movement is defined as any type of downslope movement of earth materials under hydrological dry conditions.
+  definition_wet_mass_movement: &definition-wet-mass-movement |-
+    A wet mass movement is defined as a type of mass movement that occur when heavy rain or rapid snow/ice melt send large amounts of vegetation, mud, or rock down a slope driven by gravitational forces.
+  definition_glacial_lake_outburst_flood: &definition-glacial-lake-outburst-flood |-
+    Glacial lake outburst floods are defined as those that occur when water held back by a glacier or moraine is suddenly released. Glacial lakes can be at the front of the glacier (marginal lake) or below the ice sheet (sub-glacial lake).
+  definition_fog: &definition-fog |-
```

Comment:
I don't think that's relevant here, since the only fog disaster is the Great Smog of London in 1952. To clarify, I've actually just added the sentence " Currently, the only fog disaster recorded in EM-DAT is the Great Smog of London in 1952.".